action.yml

```
@@ -23,6 +19,10 @@ inputs:
     required: false
     default: '${{ github.token }}'
     description: 'GitHub Token (available as `secrets.GITHUB_TOKEN`)'
+  rawBody:
```

nit

`rawBody` sounds like you are passing in a body

```suggestion
  useRawBody:
```

```suggestion
  isRawBody:
```